OpenSim/Simulation/SimbodyEngine/Joint.cpp

```diff
@@ -224,6 +224,18 @@ const Coordinate& Joint::getCoordinate() const {
     return get_coordinates(0);
 }
 
+Coordinate& Joint::updCoordinate() {
+    OPENSIM_THROW_IF(numCoordinates() == 0,
+                     JointHasNoCoordinates);
+    OPENSIM_THROW_IF(numCoordinates() > 1,
+                     InvalidCall,
+                     "Coordinate set has more than one coordinate. Use "
+                     "updCoordinate method defined in the concrete class "
+                     "instead.");
+
```

Comment:
Maybe good to say "*Joint* ~~Coordinates set~~ has more than one coordinate. ......".